### PR TITLE
Android/new resource structure

### PR DESCRIPTION
**Pull request by kursatbakis**
Path detail pages have the tab 'Resources' from now on. Enrolled or finished users can add the links they used during the process of learning. Other users are going to see those resources. Interactivity and helpfulness of application are going to be boosted.

### CODE REVIEW

File: source/android/Renaissance/lib/add_resource_page.dart [hunk 1]
```diff
@@ -0,0 +1,87 @@
...
+
+class AddResourcePage extends StatefulWidget {
+  @override
+  State<StatefulWidget> createState() => _AddResourcePageState();
+
+}
+
+class _AddResourcePageState extends State<AddResourcePage> {
+  TextEditingController descriptionController = TextEditingController();
+  TextEditingController linkController = TextEditingController();
```
Comment (Hazerank) [inline source/android/Renaissance/lib/add_resource_page.dart:14]: I think using local scoped variables via underscore ( like _descriptionController) is a better approach for these controllers

File: source/android/Renaissance/lib/models/user.dart [hunk 2]
```diff
@@ -15,6 +17,7 @@ class User {
 
   static User? me;
   factory User.fromJson(Map<String, dynamic> json) {
+    log(json.toString());
```
Comment (Hazerank) [inline source/android/Renaissance/lib/models/user.dart:20]: You can remove logging lines.

**Review by Hazerank — CHANGES_REQUESTED** [review]
You have done a marvelous job there. I just gave some minor reminders in here and there. Overall, that is pretty nice coding here.

**Review by fihlie — APPROVED** [review]
With these feautre our platform will be true community-driven self development platform. Can't wait to see the final form.

**Review by Hazerank — APPROVED** [review]
Thanks for your fast updates my dear colleague. I wish you a nice evening for your remaining work !

**Pull request merged**